doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+:::
+
+## Global and Local Insights
+
+Insights can be categorised into two types. Local and global. Intuitively, a local insight says something about a single prediction that a model makes. For example, given an image classified as a cat by a model, a local insight might give the set of features (pixels) that need to stay the same for that image to remain classified as a cat. Such an insight provides an idea of what the model is looking for when classifying a specific instance into a particular class.
+
+On the other hand, global insights refer to the behaviour of the model over a set of inputs. Plots showing how a regression prediction varies for a given feature while factoring out all the others are examples. These insights provide a more general understanding of the relationship between inputs and model predictions.
+
+__TODO__:
+- Add image to give idea of Global and local insight
+
+## Insights
+
+Alibi provides several local and global insights with which to explore and understand models. The following gives the practitioner an understanding of which explainers are suitable in which situations.
+
+### Local Necessary Features:
+
+Given a single instance and model prediction, local necessary features are local explanations that tell us what minimal set of features needs to stay the same so that the model still gives the same or close prediction.
+
+In the case of a trained image classification model, local necessary features for a given instance would be a minimal subset of the image that the model uses to make its decision. A machine learning engineer might use this insight to see if the model concentrates on the correct features. Local necessary features are particularly advantageous for checking erroneous model decisions.
+
+The following two explainer methods are available from alibi for generating Local Necessary Features insights. Each approaches the idea in slightly different ways. The main difference is that anchors give a picture of the size of the area over the dataset for which the insight applies, whereas pertinent positives do not.
+
+#### Anchors
+
+| Model-types | Task-types     | Data-types  |
+| ----------- | -------------- | ----------- |
+| Black-box   | Classification | Tabular     |
+|             |                | Text        |
+|             |                | Image       |
+|             |                | Categorical |
```

Comment:
For this table and the following similar ones: does it not make more sense to just have 3 bullet points instead of a table since it would always be very "unbalanced" and a bit odd-looking? E.g.:

**Alibi supports**
- Model: black-box
- Task: classification
- Data: tabular, text, image, categorical

And even for each (data) type directly link to relevant examples.

Reply:
Not sure, I feel table format feels more readable personally.

Reply:
Agree linking to examples.

Reply:
I personally prefer the table format. I've changed it so that it's only two rows like so: 

| Explainer  | Scope  | Model types  | Task types      | Data types                           | Use                                                                                             |
| ---------- | ------ | ------------ | --------------- | ------------------------------------ | ----------------------------------------------------------------------------------------------- |
| Anchors    | Local  | Black-box    | Classification  | Tabular, Categorical, Text and Image | Which set of features of a given instance is sufficient to ensure the prediction stays the same |

I've also decided to incorporate the examples into each explainer section as it feels a bit weird to have a separate section going through all the examples at the end. This can be revised though.